package.json

```diff
@@ -4,7 +4,7 @@
   "description": "IANA Uniform Resource Identifier (URI) Schemes list, including crowd sourced unofficial ones",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "./node_modules/.bin/mocha test",
```

Comment:
`./node_modules/.bin/` is unnecessary here, `npm run` and `npm test` adds it to your $PATH automatically.